client/state/cancellation-offers/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface CancellationOffer {
+	currencyCode: string;
+	discountPercentage: number;
+	discountedPeriods: number;
+	formattedPrice: string;
+	originalPrice: number;
+	rawPrice: number;
+}
+
+export interface CancellationOfferAPIResponse {
+	currency_code: string;
+	discount_percentage: number;
+	discounted_periods: 1;
```

Comment:
Could you clarify why we have a value instead of a type here?

Reply:
🤔 Not actually sure 😂 Seems that's an error, nice catch.